app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx

```diff
@@ -46,12 +47,15 @@ const messages = defineMessages({
   },
 });
 
-@connect((state) => ({
-  logItem: activeLogSelector(state),
-  historyItems: historyItemsSelector(state),
-  retryItemId: activeRetryIdSelector(state),
-  retries: retriesSelector(state),
-}))
+@connect(
+  (state) => ({
+    logItem: activeLogSelector(state),
+    historyItems: historyItemsSelector(state),
+    retryItemId: activeRetryIdSelector(state),
+    retries: retriesSelector(state),
+  }),
+  { reloadLogPage: fetchLogPageData },
```

Comment:
Why are you reload the whole page? May be better to fetch just the history items?
This should reduce the time the user waits for the action to complete.
As an better alternative, we can update the necessary item locally via redux action.